install_config/aggregate_logging.adoc

```diff
@@ -320,6 +320,10 @@ xref:../install_config/install/advanced_install.adoc#configuring-node-host-label
 |`openshift_logging_fluentd_memory_limit`
 |The memory limit for Fluentd pods.
 
+|`openshift_logging_fluentd_use_journal`
```

Comment:
It seems @anpingli is right.  As such, this should probably read that along with it being deprecated, the setting is now ignored.

Reply:
This VAR and any reference to it can be removed completely.  Grep'ing through the code I see no reference to it other then warning and debug messages that we shouldn't use it.